man/jobsub_api.md

```diff
@@ -0,0 +1,172 @@
+    * job_info -- script to call with jobid and command line upon submission
+    * L -- log file name for job output
+    * maxConcurrent -- maxumum number of jobs to run simultaneously
+    * memory -- amount of memory to request allows suffixes from KB,MB,GB,TB
+    * need_scope -- scopes needed in job auth tokens
+    * need_storage_modify -- paths to ave storage:modif in auth token scope
+    * N -- number of jobs to submit
+    * no_env_cleanup -- do not clean environment in wrapper script
+    * OS -- operating system to request, can be multiples comma separated
+    * overwrite_condor_requirements -- requirements to replace standard ones
+    * project_name -- name of SAM project to use in DAGS
+    * resource_provides -- request specific resources
+    * role -- token role to use
+    * r -- experiment release version
+    * singularity_image -- cvmfs path to singularity image for job
+    * site -- comma separated list of sites to use
+    * skip_check -- skip checks done by default from rcds
+    * subgroup -- subgroup with role for permissions
+    * tarball_exclusion_file -- file exculsions for tarfile generation
+    * job_info -- script to call with jobid and command line upon submission
+    * L -- log file name for job output
+    * maxConcurrent -- maxumum number of jobs to run simultaneously
+    * memory -- amount of memory to request allows suffixes from KB,MB,GB,TB
+    * need_scope -- scopes needed in job auth tokens
+    * need_storage_modify -- paths to ave storage:modif in auth token scope
+    * N -- number of jobs to submit
+    * no_env_cleanup -- do not clean environment in wrapper script
+    * OS -- operating system to request, can be multiples comma separated
+    * overwrite_condor_requirements -- requirements to replace standard ones
+    * project_name -- name of SAM project to use in DAGS
+    * resource_provides -- request specific resources
+    * role -- token role to use
+    * r -- experiment release version
+    * singularity_image -- cvmfs path to singularity image for job
+    * site -- comma separated list of sites to use
+    * skip_check -- skip checks done by default from rcds
+    * subgroup -- subgroup with role for permissions
+    * tarball_exclusion_file -- file exculsions for tarfile generation
+    * timeout -- end job if it runs longer than this units from ms,m,h,d
+    * t -- experiment test-relesase directory
+    * verbose -- verbosity, interger from 1 to 10
+
+## Submitted Job Objects
+
+Job objects  (existing in condor.py) have members
+* id for jobids
+* seq -- job sequence number / cluster_id
+* cluster -- boolean whether we are a whole cluster n@schedd or job n.m@schedd
+* proc -- process within cluster
+* schedd -- schedd job was submitted to
+
+SubmittedJob objects add
+* pool
+* group
+* role
+* auth_methods
+* submit_output (onlyt if the result of submit())
+* owner  (if jobsub_q output or q() methods called)
+* submitted (datetime.datetime) ( "" )
```

Comment:
This and the next two lines are wrong.  The default values, according to jobsub_api.py, are `None`, not `""`.

Reply:
Actually that was supposed to be "ditto marks" , meaning repeat the " (if jobsub_q output or q() methods called)"...  Sigh.